### PR TITLE
Fixes signing prepared message and test vectors with wrong salt.

### PR DESCRIPTION
**Pull request by armfazh**
Fixes signing prepared message and test vectors with wrong salt.

### CODE REVIEW

File: draft-irtf-cfrg-rsa-blind-signatures.md [hunk 1]
```diff
@@ -1088,142 +1228,230 @@ a4fcf1e5ccee35ad85ecd453182df9ed735893f830b570faae8be0f6fe2e571a4e0d
...
+ced52d9a7d9dc71d128c888a714f36d1f7ceb9fce2e28bf6fd1cd37a24f10755fc2d
+220113d61ec30ad245f62eed07a3252e3ae67879afc8cb956183bc2ccad23e112311
+0b57ef0b25cbc408f7cefb15978e90a10a66ce8c13458ceaa220437421a11c9f04b9
+b478fb39732f7d39c18b9bbdf47478ab8d820e85363100043f313c1a0016723d3eb6
+2b521421e9c9388584c9a46dc67c76a8e13c53aefc2435bc8059a8ddec697638cf88
+dd606dc00eced20021c7d33a631573c7ab4b1b6e489714fe5dc7a6d6b64e330c12af
+0e463d59f7993fbaca7d9b39d5f12d1b8289953e74
+~~~
+
+## new deterministic test vector Test Vector
```
Comment (chris-wood) [inline draft-irtf-cfrg-rsa-blind-signatures.md:1382]: This test vector shouldn't be included.

File: draft-irtf-cfrg-rsa-blind-signatures.md [hunk 2]
```diff
@@ -1088,142 +1228,230 @@ a4fcf1e5ccee35ad85ecd453182df9ed735893f830b570faae8be0f6fe2e571a4e0d
...
+7e87f259e250b16a9912ce2c085cb9489846f803fd6ed09bf8605c4aa8b0ebf2c938
+093e53ad025a48b97f7975255805118c33fa0f73ec204b9723acefacd8031ab3d9f7
+ebeaf996eee3678c788cea96932dd723b236355c0e6864fad2fc87b00e4eda476e90
+f000936b0d9fa65bf1112fc296e8aa5bb05ca7cb32dec01407e3d3ed94c1ebb0dc43
+0ea59588ccc0995a6e2f1423dbe06c6f27650b23b12eb343b9e461ba532825e5e265
+72fbe723b69753c178361e7a834a566ce950df55ff97d314b384b3fa8c0098d560d4
+c6ba519a9b6040f908adf34f6b2d5d30c265cd0fb1
+~~~
+
+## new probabilistic test vector Test Vector
```
Comment (chris-wood) [inline draft-irtf-cfrg-rsa-blind-signatures.md:1306]: This test vector shouldn't be included.

File: draft-irtf-cfrg-rsa-blind-signatures.md [hunk 3]
```diff
@@ -1088,142 +1228,230 @@ a4fcf1e5ccee35ad85ecd453182df9ed735893f830b570faae8be0f6fe2e571a4e0d
 3fb6f63a24
 ~~~
 
-## RSABSSA-SHA384-PSSZERO-Deterministic Test Vector
+## deterministic Test Vector
```
Comment (chris-wood) [inline draft-irtf-cfrg-rsa-blind-signatures.md:1231]: This test vector shouldn't be included.

**Review by chris-wood — CHANGES_REQUESTED** [review]
Nice! We need to drop two test vectors, but otherwise this is good.

File: draft-irtf-cfrg-rsa-blind-signatures.md (hunk 3 above)
Comment (armfazh) [reply draft-irtf-cfrg-rsa-blind-signatures.md]: done

**Review by chris-wood — APPROVED** [review]

**Pull request merged**